coinlib/lib/src/secp256k1/secp256k1_io.dart

```diff
@@ -19,8 +19,8 @@ String _libraryPath() {
     localLib = "lib$_name.dylib";
     flutterLib = "$_name.framework/$_name";
   } else if (Platform.isWindows) {
-    // Not provided yet, so should fail to load unless added outside of library
-    flutterLib = localLib = "$_name.dll";
+    // Built in build\windows.
+    flutterLib = localLib = "windows/$_name.dll";
```

Comment:
Is this `windows/` folder/prefix required to load on flutter? If so, why not include it for the `flutterLib` but keep the pure-dart library location in `build/` only to keep consistency with the other builds?

Reply:
Yes, the addition of the `windows` subfolder was required in order to use `coinlib_flutter` in another flutter project.  coinlib_flutter's own example runs fine in either location, but my consumer needed it in `build/windows/x64`

Reply:
OK, can't the `localLib` be without the `windows` folder though?

Reply:
I'll have to test this again, I thought this change was consistent with the .dll placement in build/windows.

Reply:
Resolved by 91f0f7f0e66e6ded1f58715ba5d5653d2fbb9402

Puts secp256k1.dll in /build instead of /build/windows or /build/windows/x64